pkg/tasks/c1api/list_ticket_schemas.go

```diff
@@ -48,9 +50,16 @@ func (c *listTicketSchemasTaskHandler) HandleTask(ctx context.Context) error {
 
 		ticketSchemas = append(ticketSchemas, schemas.GetList()...)
 
+		// Only return first 100 elements
+		if len(ticketSchemas) >= maxTicketSchemas {
+			ticketSchemas = ticketSchemas[:maxTicketSchemas]
```

Comment:
It'd be nice to log if we hit this condition, and if so, how many ticket schemas we ignored.

Reply:
ill add a log, the number of ticket schemas we ignore could be more though if theres additional pages